src/opera/pge/rtc_s1/rtc_s1_pge.py

```diff
@@ -710,6 +716,40 @@ def _collect_rtc_product_metadata(self, metadata_product):
         output_product_metadata['data']['length'] = len(output_product_metadata['data']
                                                         ['yCoordinates'])
 
+        product_type = self.runconfig.product_type
+
+        # If creating ISO metadata for a static layer product, we need to
```

Comment:
I think this comment needs to be expanded, particularly since we now have 2 ways of extracting data to determine the bounding boxes.